util/string_helpers.py

```diff
@@ -64,3 +65,22 @@ def random_string(size):
     :return: A Unicode string.
     """
     return binascii.hexlify(os.urandom(size)).decode("utf8")
+
+
+def native_string(x):
```

Comment:
[The porting guide](https://portingguide.readthedocs.io/en/latest/strings.html#the-native-string) talks about the "native string" type, a.k.a. `str`, which represents Unicode data in Python 3 and byte data in Python 2. Usually we can avoid this data type by being explicit, but the `__repr__` method needs to return this type, which led to a lot of PYTHON3 TODOs.

But thinking about it that way, I figured I could get rid of a lot of those TODOs with a helper function that turns... some kind of string... into... the native string type for the current Python version.